.gitattributes

```diff
@@ -7,8 +7,8 @@
 *.json text eol=lf whitespace=blank-at-eol,blank-at-eof,space-before-tab,tab-in-indent,tabwidth=4
 *.yml text eol=lf whitespace=blank-at-eol,blank-at-eof,space-before-tab,tab-in-indent,tabwidth=4
 
-# don't add these file to github release tarballs (composer dist install)
-/src/ISO3166Test.php export-ignore
+# don't add these files/directories to github release tarballs (composer dist install)
+/tests/ export-ignore
```

Comment:
I know, but this is a subject where we strongly differ in opinion. There is no middle ground unfortunately :-)